measurements/api/measurements.py

```diff
@@ -149,15 +122,14 @@ def api_list_report_files():
             'probe_asn': "AS{}".format(row.probe_asn),
             'test_name': row.test_name,
             'index': int(row.report_no) + REPORT_INDEX_OFFSET,
-            'test_start_time': row.test_start_time.strftime('%Y-%m-%dT%H:%M:%SZ')
+            'test_start_time': row.test_start_time
```

Comment:
Why?

Reply:
Ah, because now you've implemented a flask filter to automatically encode into JSON.

Reply:
yup